src/recoil/big_dipper_networks/hooks.ts

```diff
@@ -14,13 +14,13 @@ import {
 import {
   useChainIdQuery,
   ChainIdQuery,
-} from '@graphql/types';
+} from '@graphql/types/general_types';
 import {
   Networks,
   Selected,
 } from '@recoil/big_dipper_networks/types';
 
-const NETWORK_LIST_API = 'https://raw.githubusercontent.com/forbole/big-dipper-networks/main/networks.json';
+const NETWORK_LIST_API = 'https://raw.githubusercontent.com/cheqd/big-dipper-networks/cheqd/cheqd-networks.json';
```

Comment:
@nikola-cheqd maybe we can have this as an env var as well? just so that we keep everything generalised?